include/psf_config.h

```diff
@@ -53,6 +53,7 @@ namespace psf
 
     struct json_value
     {
+        virtual ~json_value() {}

```

Comment:
What is the destructor for?  PSF worked without it.

Reply:
I would say to not include a destructor until it has custom logic.

Reply:
The new version of the windows SDK is throwing warning in case where the destructor is missing. Moreover it make sense to have the virtual destructor here and we are mandated to have a definition attached with the virtual destructor as there are never going to be overridden.

Reply:
base class needs to have a virtual destructor for derived to clean up properly if delete is called on a base pointer. This error is likely from the toolset change. Either way this is goodness.